test/cluster/mv/test_mv_topology_change.py

```diff
@@ -256,6 +256,11 @@ async def do_writes() -> int:
 @pytest.mark.asyncio
 @pytest.mark.parametrize("delayed_replica", ["base", "mv"])
 @pytest.mark.parametrize("altered_dc", ["dc1", "dc2"])
+# FIXME: The test relies on cross-rack tablet migrations. They're forbidden when the configuration option
```

Comment:
Is RF change currently supported with `rf_rack_valid_keyspaces` on? If no there should be an issue for it, which should be mentioned here (you can also use xfail mark).